docs/run-random-beacon.adoc

```diff
@@ -6,6 +6,46 @@
 :numbered:
 toc::[]
 
+== Staking
+
+=== Terminology
```

Comment:
I would actually move this to the end <_<

Reply:
On it.

Reply:
Eh, I actually wonder if this whole section shouldn't be its own doc linked from here. This doc is about operating the client 🤔

Reply:
Yeah, I think we should give https://github.com/keep-network/keep-core/pull/1455 more attention though I think it makes sense to have it here temporarily before we can get staking doc out. What do you think?

Reply:
Yeah that's probably fair. At least it'll give us something to point at.

Reply:
Moved to the bottom.